### PR TITLE
improve commandline examples

### PR DESCRIPTION
**Pull request by frusen**
I had to specify a language (java worked) for some code blocks to make the formatting ok. Try removing it to see what I mean. Maybe there is a way to specify no formatting.

### CODE REVIEW

File: docs/handbook/getting-started.md [hunk 1]
```diff
@@ -281,17 +307,18 @@ If you wish to purchase a ready made DVD or USB drive there is also [OSDISC.COM]
...
 
 ### Checking the MD5/SHA
 
 Checksum verification example:
 
 ```
-sha256sum --check OI-hipster-gui-20161030.iso.sha256sum
+$ sha256sum --check OI-hipster-gui-20161030.iso.sha256sum
 OI-hipster-gui-20161030.iso: OK
+$
```
Comment (pyhalov) [inline docs/handbook/getting-started.md]: Is this line intentional?

**Comment by pyhalov** [comment]
Should PKG section be also updated?

File: docs/handbook/getting-started.md (hunk 1 above)
Comment (frusen) [inline docs/handbook/getting-started.md]: Yes, to show that the prompt has returned. Did it in a few other places as well.

**Comment by frusen** [comment]
@pyhalov yes seems like i missed that one.

File: docs/handbook/getting-started.md (hunk 1 above)
Comment (pyhalov) [inline docs/handbook/getting-started.md]: OK

File: docs/handbook/getting-started.md [hunk 2]
```diff
@@ -38,6 +38,32 @@ This collection of documents is aimed at providing a comprehensive source of gui
 While our aim is to address as many facets of OpenIndiana use and administration as possible, some topics are simply too complex and beyond the scope of an introductory end user handbook.
 In such cases external sources of information will be provided in the form of references to other web sites, man pages, or printed books.
 
+## Conventions
+
+### Commandlines
+
+Commandlines starting with a # character means that they should be run as root.
```
Comment (pyhalov) [inline docs/handbook/getting-started.md]: Why not "Command lines"?

Comment (frusen) [inline docs/handbook/getting-started.md]: "Command lines" or "command-lines" could work instead. Examples is another alternative.

Comment (pyhalov) [inline docs/handbook/getting-started.md]: command lines mean ?

Comment (xen0l) [inline docs/handbook/getting-started.md]: Command examples might be more appropriate (as in http://dtrace.org/guide/preface.html#fwbsd)

**Comment by makruger** [comment]
looks like MDL is complaining a bit. We can probably disable the rule for dollar signs, but adding some whitespace after code block fencing will fix the other. Thank you for the help.

mdl -s markdownlint-rules.rb .

./docs/handbook/getting-started.md:63: MD014 Dollar signs used before commands without showing output

./docs/handbook/getting-started.md:203: MD014 Dollar signs used before commands without showing output

./docs/handbook/getting-started.md:215: MD014 Dollar signs used before commands without showing output

./docs/handbook/getting-started.md:221: MD014 Dollar signs used before commands without showing output

./docs/handbook/getting-started.md:309: MD014 Dollar signs used before commands without showing output

./docs/handbook/getting-started.md:471: MD031 Fenced code blocks should be surrounded by blank lines

./docs/handbook/getting-started.md:659: MD031 Fenced code blocks should be surrounded by blank lines

**Comment by makruger** [comment]
Was looking into disabling markdownlint rule MD014 and found the following:

https://github.com/mivok/markdownlint/blob/master/docs/RULES.md#md014---dollar-signs-used-before-commands-without-showing-output

Also see:

http://www.cirosantilli.com/markdown-style-guide/#dollar-signs-in-shell-code

They both basically say it's bad form to include dollar signs and hashtags as it obfuscates the documentation and makes it more difficult to perform a copy/paste to the command line.

**Comment by xen0l** [comment]
@makruger what's the status?

**Comment by makruger** [comment]
Sorry I've been too busy with life to give this much consideration. I'll push through a commit excluding rule MD014, but this commit needs to be fixed to comply with rule MD031 to add some padding around code fenced code blocks.

**Pull request merged**